src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,52 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_CAPACITY = 10;
+    private Object[] keyArray = new Object[MAX_CAPACITY];
+    private Object[] valueArray = new Object[MAX_CAPACITY];
+    private int size = 0;
```

Comment:
1. Better to properly paramtrize arrays during initialization in order not to type-cast in methods
2. Better to initialize variable in constructor
3. 0 is default int value, no need to explicitly set it
4. It's very rarely makes sense to include type into variable name
```suggestion
    private K[] keys;
    private V[] values;
    private int size;
    
    public Storage() {
        keys = (K[]) new Object[MAX_CAPACITY];
        values = (K[]) new Object[MAX_CAPACITY];
    }
```